packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -39,6 +43,35 @@ export interface WaferMapColorScale {
     values: string[];
 }
 
+export interface ZoomHandlerData {
+    canvas: HTMLCanvasElement;
+    zoomContainer: HTMLElement;
+    containerDimensions: Dimensions;
+    canvasLength: number;
+    renderModule: RenderingModule;
+}
+
+export interface HoverHandlerData {
```

Comment:
I'm open to other opinions on this, but I think it would be easier to follow if each of these interfaces moved to the handler files that use them. The `types.ts` file is more useful for types that are expected to be used publicly.

Reply:
Tbh ... I also do not have a strong opinion on this. I am open to this change. The only issue is the presence of the `HoverHandlerData`, which I do not have a place yet to relocate to for this refactor and would need to be left here until the next PR or removed and re-added later ... which I would prefer not to do :) - it would impact other interfaces...

Reply:
I did apply this change, however I did leave for now the `HoverHandlerData ` within the "types" folder - I think it would be best if this change is applied in the follow-up PR once we add the actual `HoverHandler `code...